### ISSUE TITLE
Create python script for data analysis

### ISSUE THREAD

**Issue by antoniosValmas** [WraioPragma/HashCode2020#2]
Create a python script for data analysis. It should include:
- Read data from files
- Visualize data through plots and diagrams
- Perform further analysis (I am not sure what this will include)
- Save data to a database (SQLite)

Test this script on the practice round to be sure that it works and it can provide helpful information.

**Comment by mpourmpoulis** [comment]
Lacking the specific format of the problem, I think for the time being we can only create a skeleton of what we want, For instance how we want The command line interface to be, Some generic code to interface with the database, for example Generic methods to create tables, Wrappers around insertions In a user-friendly manner And so on

**Comment by antoniosValmas** [comment]
Yes that was my plan and we can test that this skeleton is versatile enough by testing it with the practice round

**Comment by mpourmpoulis** [comment]
Αν δεν προλαβαινεις, θα μπορουσα ισως να το κανω εγω, αλλα θα ειμαι φουλ πιεσμενος αυτες τις δυο βδομαδες. Θες μηπως να ασχοληθουμε απο κοινού και το προχωραμε σιγα σιγα?
